src/qt/scicon.cpp

```diff
@@ -11,6 +11,7 @@
 #include <QPalette>
 #include <QPixmap>
 
+#if !defined(WIN32) && !defined(MAC_OSX)
 static void MakeSingleColorImage(QImage& img, const QColor& colorbase)
 {
     img = img.convertToFormat(QImage::Format_ARGB32);
```

Comment:
What about wrapping L16-L24 with the #ifdefs and remove the ifdef at L30 below?
This will probably remove the warning, keep the amounts of ifdefs.